server/plugin.go

```diff
@@ -401,17 +401,17 @@ func (p *Plugin) handleCreateMeetingInServerVersion(w http.ResponseWriter, r *ht
 		},
 	}
 
-	if post, err := p.API.CreatePost(post); err != nil {
-		mlog.Error("Error creating a new post with the new meeting: " + err.Error())
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	post, appErr := p.API.CreatePost(post)
+	if appErr != nil {
+		mlog.Error("Error creating a new post with the new meeting: " + appErr.Error())
+		http.Error(w, appErr.Error(), http.StatusInternalServerError)
+		return
+	}
+
+	if appErr = p.API.KVSet(fmt.Sprintf("%v%v", POST_MEETING_KEY, newMeetingResponse.MeetingId), []byte(post.Id)); appErr != nil {
```

Comment:
Looks great! I just have one question: maybe this `appErr != nil {` should be in a new line to be more readable?

Reply:
Sure. Done